### ISSUE TITLE
Dashboard: Labels cannot be programmatically associated to multiple input fields.

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#14]
**URL**: https://hcdev.openplus.ca/en/user/197
**User agent**: Chrome/Firefox
**Language**: both

**Issue Description**: Some form fields have multiple form labels. For example, Title field is 4 times on the page and all labels are Title and the input fields have id as "edit-title". 

**Steps to reproduce**: WAVE toolbar, source code

**Expected results**: 
Some form controls have more than one label associated with them. 4 in this case for each of the fields in each section. A form control should have at most one associated label element. 
Screen reader support enabled.

**Comment by davidgotlieb** [comment]
This has to do with the artificial scenario we created for testing, where the account that was being used was created with all user roles. Openplus indicates it would not occur in real scenario. At most it's very much a fringe use case that would be very unlikely to occur.

**Comment by qanitah** [comment]
The ids have been updated and the reasoning also makes sense that the user will not see all scenarios ( hopefully will see only 1 at a time).
